<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Крестики Нолики</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Image/Image.qrc">
    <normaloff>:/new/prefix1/icon.png</normaloff>:/new/prefix1/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>471</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Игра</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_1">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QPushButton" name="pushButton_9">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="pushButton_8">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="pushButton_7">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Об игре</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Made by Evgeny Khokhlov</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Image/Image.qrc">:/new/prefix1/Lable.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>281</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_O">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>O</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_start">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Играть</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_X">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_message">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_about">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Об игре</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Image/Image.qrc"/>
 </resources>
 <connections/>
</ui>
